create.sh

```diff
@@ -170,6 +172,41 @@ elif [ "$TEMPLATE_KEY" = "php" ]; then
     _composer="php composer.phar"
   fi
   install_command="$_composer install"
+
+#### PHP LARAVEL ####
+elif [ "$TEMPLATE_KEY" = "php-laravel" ]; then
+  check_for_command "php"
+
+  if type -p composer > /dev/null; then
+    _composer=composer
+  else
+    php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+    php composer-setup.php
+    php -r "unlink('composer-setup.php');"
+    _composer="php composer.phar"
+  fi
+
+  # Generate application key
+  php artisan key:generate
+  
+  install_command="$_composer install && php artisan key:generate"
+
+#### PHP LEAF ####
+elif [ "$TEMPLATE_KEY" = "php-leaf" ]; then
+  check_for_command "php"
+
+  if type -p composer > /dev/null; then
+    _composer=composer
+  else
+    php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+    php composer-setup.php
+    php -r "unlink('composer-setup.php');"
+    _composer="php composer.phar"
+  fi
+  
+  install_command="$_composer install"
```

Comment:
it looks like this setup is the same as plain php? if so we could just update that condition to check for `php` or `php-leaf`

Reply:
You're absolutely right! I have fixed it now I think...